app/controllers/admin/publishers_controller.rb

```diff
@@ -50,6 +50,12 @@ def show
     @potential_referral_payment = @publisher.most_recent_potential_referral_payment
     @referral_owner_status = PromoClient.owner_state.find(id: params[:id])
     @current_user = current_user
+
+    if payout_in_progress? || Date.today.day < 12 # Let's display the payout for 5 days after it should complete (on the 8th)
+      @payout_report = PayoutReport.where(final: true, manual: false).order(created_at: :desc).first
+      @payout_message = PayoutMessage.find_by(payout_report: @payout_report, publisher: @publisher)
```

Comment:
Duplicate call?

Reply:
One if PayoutReport the other is PayoutMessage 😄